Server/controllers/User/logIn.js

```diff
@@ -0,0 +1,88 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+const { validateEmail } = require('../../helpers/Validation');
+
+const { SECRET } = process.env;
+const { logIn } = require('../../database/query/User/index');
+const {
+  successMessage,
+  internalErrorMessage,
+  faildLoginMessage,
+  faildMessage,
+} = require('../../helpers/responseMessage');
+
+// To create user token
+const createToken = (email, gid, display_name, secret) =>
+  sign({ email, gid, display_name }, secret);
+
+module.exports = (req, res) => {
+  const { email, password } = req.body;
+  const { error } = validateEmail({ email });
+
+  // If user type his/her email incorrectly
+  if (error !== undefined) {
+    return res
+      .status(403)
+      .json(
+        faildLoginMessage(
+          null,
+          'Make sure that you type your email in a correct way',
+        ),
+      );
+  }
+
+  // If user doesn't enter his/her email or password or both of them
+  if (!email || !password) {
+    return res.status(400).json(faildMessage(null, 'All Fields are Required!'));
+  }
+
+  // TO get password for the entered email from database
+  logIn(email)
+    .then((result) => {
+      // If the email isn't in the database
+      if (!result || result.rows.length === 0) {
```

Comment:
your code is good, but I have some advice and you have the freedom to apply this advice or not

1. `!result` it is not necessary because when the code enters in  `then` meaning the code run successfully and the result Absolutely have value.
the `result`  is undefined just when the error happened, and this case handled in the catch  

2. `result` contains `rowCount` property you can use it directly and it faster